tests/phpunit/tests/functions/wpGetDirectUpdateHttpsUrl.php

```diff
@@ -0,0 +1,44 @@
+<?php
+
+/**
+ * Tests for the wp_get_direct_update_https_url() function.
+ *
+ * @group functions.php
+ * @covers ::wp_get_direct_update_https_url
+ */
+class Tests_Functions_wpGetDirectUpdateHttpsUrl extends WP_UnitTestCase {
+
+	/**
+	 * Test that wp_get_direct_update_https_url() returns the correct URL when WP_DIRECT_UPDATE_HTTPS_URL is set.
+	 */
```

Comment:
```suggestion
	 *
	 * @ticket 59623
	 */
```
:1234: Applies to all tests in the PR.